python/prodV4/worker.py

```diff
@@ -0,0 +1,74 @@
+import os
+import sys
+
+from pathlib import Path
+
+# User packages
+PKGS_PATH = "site/wwwroot/.python_packages"
+VENV_PKGS_PATH = "site/wwwroot/worker_venv"
+
+PKGS = "lib/site-packages"
+
+# Azure environment variables
+AZURE_WEBSITE_INSTANCE_ID = "WEBSITE_INSTANCE_ID"
+AZURE_CONTAINER_NAME = "CONTAINER_NAME"
+AZURE_WEBJOBS_SCRIPT_ROOT = "AzureWebJobsScriptRoot"
+
+
+def is_azure_environment():
```

Comment:
nit: Look like v2 and v3 worker files have same functions and similar implementations, shall we refactor them into worker abstract class with abstract/hook methods?

Reply:
As the worker.py is the main file is called, we do want it to be self-contained.